docs/book/component-gallery/secrets-managers/secrets-managers.md

```diff
@@ -31,11 +35,25 @@ is also a security risk, because it basically represents a large attack surface.
 With centralized secrets management, you only need to configure the ZenML server
 to access the cloud back-end.
 
-ZenML currently only supports a SQL database as a centralized secrets store, but
-we are working on re-purposing all existing secrets manager flavors (AWS, GCP,
-Azure and HashiCorp Vault) to act as back-ends for the centralized secrets
-store. As soon as this is available, we will deprecate the secrets manager stack
-component and provide an upgrade path for existing users.
+ZenML currently supports configuring the ZenML server to use the following
+back-ends as centralized secrets store replacements for secrets managers:
+
+* the SQL database that the ZenML server is using to store other managed objects
+such as pipelines, stacks, etc. This is the default option and replaces the
+`local` secrets manager flavor.
+* AWS Secrets Manager - replaces the `aws` secrets manager flavor.
+* GCP Secret Manager - replaces the `gcp` secrets manager flavor.
+* Azure Key Vault - replaces the `azure` secrets manager flavor.
+* HashiCorp Vault - replaces the `vault` secrets manager flavor.
+
+The centralized secrets store also supports using a custom back-end
+implementation.
+
+There is no direct migration path planned for the GitHub secrets manager flavor,
+given that it can only be used inside a GitHub Action workflow and thus is not
+a service that can be used as a centralized secrets store.
+If you are using the GitHub secrets manager flavor, you have the option of
+migrating your secrets to one of the other supported secrets store back-ends.
```

Comment:
```suggestion
manually transferring your secrets to one of the other supported secrets store back-ends.
```
'migrating' seems a bit weasly, given that we're telling them that we do not help them with that...